x.py

```diff
@@ -216,16 +216,18 @@ def write_version(release_version: str) -> str:
     return version
 
 
-def package_source(release_version: str) -> None:
+def package_source(release_version: str, release_candidate_number: Optional[int]) -> None:
     # 0. Write input version to VERSION file
     version = write_version(release_version)
 
     # 1. Git commit and tag
     git = find_command('git', msg='git is required for source packaging')
     run(git, 'commit', '-a', '-m', f'[source-release] prepare release apache-kvrocks-{version}')
-    run(git, 'tag', '-a', f'v{version}', '-m', f'[source-release] copy for tag v{version}')
-
-
+    if release_candidate_number == None:
```

Comment:
```suggestion
    if release_candidate_number is None:
```

Pythonic and IIRC `==` here can be problematic.